docs/products/kafka/howto/tiered-storage-overview-page.md

```diff
@@ -1,72 +1,75 @@
 ---
 title: Tiered storage overview page in Aiven Console
 sidebar_label: Tiered storage overview page
-early: true
+
 ---
 
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+import {ConsoleIcon} from "@site/src/components/ConsoleIcons"
+
 Aiven for Apache Kafka® offers a comprehensive overview of tiered storage, allowing you to understand its usage and make informed decisions.
 
 This overview page provides insights into various aspects of tiered storage, including
 hourly billing, settings, and storage details.
 
 ## Prerequisite
 
--   [Tiered storage enabled for the Aiven for Apache Kafka service](/docs/products/kafka/howto/enable-kafka-tiered-storage).
+- [Tiered storage enabled for the Aiven for Apache Kafka service](/docs/products/kafka/howto/enable-kafka-tiered-storage).
 
 ## Access tiered storage overview page
 
-1.  Log in to the [Aiven console](https://console.aiven.io/), choose
-    your project and select your Aiven for Apache Kafka service.
-1.  From the left sidebar, select **Tiered Storage**.
-    -   If tiered storage is not yet enabled for your service, you will
-        see the option to enabled it.
-    -   If tiered storage is enabled but not configured for any topics,
-        you have the option to set it up for topics directly. For more
-        details, see
-        [Enable and configure tiered storage for topics](/docs/products/kafka/howto/configure-topic-tiered-storage).
-1.  Once configured, you can view an overview of tiered storage and its
-    associated details.
+1. Log in to the [Aiven console](https://console.aiven.io/), choose
+   your project and select your Aiven for Apache Kafka service.
+1. Click <ConsoleLabel name="Tiered storage" />.
+
+   -   If tiered storage is not yet enabled for your service, you will
+       see the option to enabled it.
+   -   If tiered storage is enabled but not configured for any topics,
+       you have the option to set it up for topics directly. For more
+       details, see
+       [Enable and configure tiered storage for topics](/docs/products/kafka/howto/configure-topic-tiered-storage).
+1. Once configured, you can view an overview of tiered storage and its
```

Comment:
Is this step actionable? If clicking "Tiered storage" displays the overview page, it seems not actionable.
How about including it as plain text, not a step?
E.g. Once tiered storage is enabled and configured, the tiered storage overview page is displayed upon clicking **Tiered storage**? or something similar.

Reply:
If tiered storage is enabled at the service level but not at the topic level, clicking "Tiered storage" to view the overview prompts the user to activate it. This process is described in the steps provided. Removing this step would omit critical information that guides the user through the necessary actions.